tox.ini

```diff
@@ -23,9 +23,8 @@ deps =
   flake8
   flake8-import-order==0.9.2
   pep8-naming
-
 commands =
-  flake8 --verbose {env:CI_FLAGS:} dcos tests setup.py
+  flake8 --verbose {env:CI_FLAGS:} cli dcos tests setup.py
```

Comment:
This is already tested in the other tox file (cli/tox.ini). We have two because we publish two packages when we release (dcos and dcoscli package).